src/wombats/daos/access_key.clj

```diff
@@ -0,0 +1,35 @@
+(ns wombats.daos.access-key
+  (:require [datomic.api :as d]
+            [wombats.daos.helpers :refer [get-entities-by-prop
+                                          get-entity-by-prop
+                                          retract-entity-by-prop]]))
+
+(defn get-access-keys
+  [conn]
+  (fn []
+    (get-entities-by-prop conn :access-key/id)))
+
+(defn get-access-key-by-id
+  [conn]
+  (fn [id]
+    (get-entity-by-prop conn :access-key/id id)))
+
+(defn get-access-key-by-key
+  [conn]
+  (fn [key]
+    (get-entity-by-prop conn :access-key/key key)))
+
+(defn add-access-key
+  [conn]
+  (fn [access-key]
+    @(d/transact conn [[:add-access-key access-key]])))
+
+(defn retract-access-key
+  [conn]
+  (fn [access-key-id]
+    @(retract-entity-by-prop conn :access-key/id access-key-id)))
+
+(defn update-access-key
+  [conn]
+  (fn [access-key-id]
+    "NOT IMPLEMENTED DUE TO MISSING REQUIREMENTS https://github.com/willowtreeapps/wombats-api/issues/341"))
```

Comment:
we talked about this earlier today - event coordinators shouldn't have the UI built for this but we should be able to increase number of people per key, delete safely (no deletion of access to the app), and extend expiration date.